dsvjetl/uhp-whos-your-governor
# Feature/create layout

### TASK

- Create styled markup without main logic.
- Make layout responsive up to 320px.

### CHANGES

- [x] Create styled markup without main logic.
- [x] Make layout responsive up to 320px.

### QA

- Make sure layout is as much similar to PDG design.
- Make sure layout is not breaking on different resolutions.
